sdk/textanalytics/ai-text-analytics/src/analyzeBatchActionsResult.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PagedAsyncIterableIterator } from "@azure/core-paging";
+import { ExtractKeyPhrasesResultArray } from "./extractKeyPhrasesResultArray";
+import { TextDocumentBatchStatistics } from "./generated/models";
+import { RecognizeCategorizedEntitiesResultArray } from "./recognizeCategorizedEntitiesResultArray";
+import { RecognizePiiEntitiesResultArray } from "./recognizePiiEntitiesResultArray";
+
+/**
+ * The results of a successful analyze batch actions operation.
+ */
+export interface AnalyzeBatchActionsResult {
+  /**
+   * Array of the results for each categorized entities recognition action.
+   */
+  recognizeEntitiesResults: RecognizeCategorizedEntitiesResultArray[];
```

Comment:
Change `RecognizeCategorizedEntitiesResultArray` to `RecognizeEntitiesResultArray`?

Reply:
I can not, the current naming has been GA'd already and I think it is the correct one.